tests/managed-service/test_post_installation_state.py

```diff
@@ -18,6 +18,10 @@ class TestPostInstallationState(ManageTest):
     Post-installation tests for ROSA and OSD clusters
     """
 
+    @managed_service_required
+    def test_post_installation(self):
```

Comment:
Shouldn't this be in libtest? It seems to be not a tier test and it won't be executed during test runs because there is no proper marker attached.

Reply:
I added this test to acceptance suite. It would be useful for the dev team, as far as I understand.

Reply:
This is the same ocs_install_verification which will be running after deployment right ?